src/pages/index.js

```diff
@@ -149,30 +100,53 @@ const SOLUTIONS = {
     }],
 
   }, {
-    title: 'Improve production consistency',
-    description: 'Use the built in scaffolder to create new services from templates which have your best practices built in.',
-    illustration: {
-      webp: ProductionConsistencyIllustration,
-      png: ProductionConsistencyIllustrationPng,
-      alt: '5 services represented by one hexagon each. The hexagons are tightly packed together in a hive-like structure and they are all the same size. This indicates that they are consistent.',
-    },
+    title: 'Standardize service creation',
+    description: 'Create services from templates which already have your best practices built in.',
     bullets: [{
-      name: 'Inconsistent environments slow development',
-      description: `Nobody wants obstacles on the path to production, but heterogeneous environments cause operational headaches and slow technology migrations.`,
+      name: 'Increase production consistency',
+      description: `Homogeneous environments are easier to operate and scale. Cut down on snowflake services.`,
       icon: FingerPrintIcon,
     }, {
       name: 'Golden path to production',
-      description:
-        `Encode your best practices in templates which service teams can use to create new services. This ensures that increasing production consistency is the path of least resistance.`,
+      description: `Encode your best practices to make production consistency the path of least resistance.`,
       icon: MapIcon,
     }],
 
+  }, {
+    title: 'Codify service ownership',
+    description: 'Track teams and software and assign one to the other.',
+    bullets: [{
+      name: 'Improve maintainability',
+      description: `Unowned services are a maintenance, security and operations nightmare.`,
+      icon: FingerPrintIcon,
+    }, {
+      name: 'Unlock InnerSourcing',
+      description: `Empower teams to contribute to the code around them. It starts with a conversation with the service owner.`,
+      icon: MapIcon,
+    }],
+
+  }, {
+    title: 'Tech docs that get read',
+    description: 'Docs-like-code helps Spotify document thousands of internal components.',
+    bullets: [{
+      name: 'Write docs as code',
+      description: 'Markdown docs live alongside the code where they can be peer reviewed in pull requests.',
+      icon: PencilAltIcon,
+    }, {
+      name: 'Organized and searchable',
+      description: 'Docs are organized in the service catalog and searchable for discoverability.',
+      icon: SearchCircleIcon,
+    }, {
+      name: 'Demonstrated at scale',
+      description: 'Spotify experienced an explosion of docs writing after adopting this methodology internally.',
+      icon: PresentationChartLineIcon,
+    }],
   }],
 };
 
 const PLUGINS_SUPPORTED = {
-  title: 'Pre-loaded with the best plugins',
-  subTitle: `Open-source plugins are supported by default. Push your own plugins if you need something bespoke.`,
+  title: 'Powerful plugins',
```

Comment:
We should probably increase the amount of plugins displayed here. There are a bunch more logos we can add.